tools/st2ctl

```diff
@@ -122,12 +122,18 @@ function register_content() {
   echo "Registering content..."
 
   if [ ! ${1} ]; then
-      REGISTER_FLAGS="--register-sensors --register-actions"
+    REGISTER_FLAGS="--register-sensors --register-actions"
+  elif [ ${1} == "-v" -o ${1} == "--verbose" ]; then
+    if [ ! ${2} ]; then
+      REGISTER_FLAGS="--register-sensors --register-actions --verbose"
```

Comment:
I think default should NOT be verbose, it's exactly the problem we are trying to solve?